.github/workflows/lfric_test.yml

```diff
@@ -75,6 +75,43 @@ jobs:
         pip install .[test]
         pip install jinja2
 
+    # PSyclone, compile and run MetOffice gungho_model on GPU
+    - name: LFRic GungHo with OpenACC offload
+      run: |
+        # Set up environment
+        source /apps/spack/psyclone-spack/spack-repo/share/spack/setup-env.sh
+        spack load lfric-build-environment%nvhpc
+        source .runner_venv/bin/activate
+        export PSYCLONE_LFRIC_DIR=${GITHUB_WORKSPACE}/examples/lfric/scripts
+        export PSYCLONE_CONFIG_FILE=${PSYCLONE_LFRIC_DIR}/KGOs/lfric_psyclone.cfg
+        export LFRIC_DIR=${HOME}/LFRic/lfric_apps_r2222
```

Comment:
I see this doesn't use the LFRIC_APPS_REV revision, and it uses an older version. Was there any problem with the newer revision? If there was a problem, is it worth documenting it here with an inlined comment and TODO if an issue exists?

Reply:
It was more that I had a patch from Lukas and this was the version of the source he was working with. You're right though that we do need to upate this so I'll add a TODO.

Reply:
I've asked Lukas if he can update his LFRic working copy and thus the patch.

Reply:
He already had a patch so I've used that and am working from the same LFRic apps revision now.